ui/css/treeherder.css

```diff
@@ -213,8 +213,13 @@ th-watched-repo {
 
 #platform-job-text-search-field {
     height:28px;
-    width: inherit;
+    width: 150px;
     display: inherit;
+    transition: width .2s;
+}
+
+#platform-job-text-search-field:focus,#platform-job-text-search-field:valid {
+    width:300px !important;
```

Comment:
I think generally we observe a character of white space here also:

`width: 300px !important;`